include/seqan3/range/detail/inherited_iterator_base.hpp

```diff
@@ -67,54 +71,74 @@ class inherited_iterator_base : public base_t
     constexpr inherited_iterator_base & operator=(inherited_iterator_base && rhs) = default;
     ~inherited_iterator_base() = default;
 
-    inherited_iterator_base(base_t it) : base_t{it} {}
+    //!\brief Delegate to base class if inheriting.
```

Comment:
```suggestion
    //!\brief Delegate to base class if inheriting from non-pointer iterator.
```

Reply:
I changed this, please mark your comments as resolved, if they are resolved.

Reply:
this is so annoying that this is not automatically resolved 😒